### PR TITLE
found in chapter 8 during translation

### PR DESCRIPTION
**Pull request by aszasz**
A bit bolder suggestions than my usual, but I think they apply. Thanks!

### CODE REVIEW

File: 08-compiler-features.tex [hunk 1]
```diff
@@ -207,20 +207,20 @@ \section{Completion}
 \end{lstlisting}
 
 \paragraph{Error Handling}
-When run in completion mode, the compiler will not display errors but instead try to ignore them or recover from them. 
+When run in completion mode, the compiler will not display errors, but instead, try to ignore them or recover from them. 
 
-If a not recoverable error occurs while getting completion, the Haxe compiler will print the error message instead of the completion output. You can then treat any not-XML output as an error message which prevents the completion.
+If a not recoverable error occurs while getting completion, the Haxe compiler will print the error message instead of the completion output. You can, then, treat any not-XML output as an error message which prevents the completion.
```
Comment (Simn) [inline 08-compiler-features.tex]: I don't think commas like this are strictly necessary, but I'm fine with it.

File: 08-compiler-features.tex [hunk 2]
```diff
@@ -228,7 +228,7 @@ \section{Completion}
 haxe --connect 6000 myproject.hxml
 \end{lstlisting}
 
-Please note that you can use \ic{--cwd} at the first sent commandline parameter to change the Haxe server's current working directory. Usually class paths and other files are relative to your project.
+Please, note that you can use the parameter \ic{--cwd} at the first sent command line to change the Haxe server's current working directory. Usually class paths and other files are relative to your project.
```
Comment (Simn) [inline 08-compiler-features.tex]: This comma should definitely go though. "Please note" belong together, if we have a comma there it sounds more like "Please, for the love of god, note that" to me.

File: 08-compiler-features.tex [hunk 3]
```diff
@@ -239,14 +239,16 @@ \section{Completion}
...
-warnings and other messages can also be considered errors, but are not fatal ones. If a fatal error occurred, we will send a single \ic{\\x02} message-line
-Here's some code that will treat connect to the server and handle the protocol details :
+Macros and other commands can log events which are not errors. From the command line, we can see the difference between what is printed on \ic{stdout} and what is print on \ic{stderr}. This is not the case in socket mode. In order to differentiate between the two, log messages (not errors) are prefixed with a \ic{\\x01} character and all newline-characters in the message are replaced by the same \ic{\\x01} character.
+
+Warnings and other messages can also be considered errors, but are not fatal ones. If a fatal error occurred, it will send a single \ic{\\x02} message-line.
+
+Here's some code that will treat connection to the server and handle the protocol details :
```
Comment (Simn) [inline 08-compiler-features.tex]: While you're at it, could you remove the extra space before the colon at the end here?

File: 08-compiler-features.tex (hunk 1 above)
Comment (aszasz) [inline 08-compiler-features.tex]: I took it back, then. You're right, It is ok without it.

**Comment by Simn** [comment]
Thanks!

File: 08-compiler-features.tex (hunk 2 above)
Comment (aszasz) [inline 08-compiler-features.tex]: You are right again, thank you. I guess it is a portuguese speaker bias. Please is translated as 'by favour', so the comma would make some sense there.

File: 08-compiler-features.tex (hunk 3 above)
Comment (aszasz) [inline 08-compiler-features.tex]: Done. Thanks

**Pull request merged**